ecs/elections/algorithms/greedy_algorithm.py

```diff
@@ -0,0 +1,87 @@
+from ecs.elections.algorithms.algorithm import Algorithm
+from ecs.utils.math import ell_p_norm
+
+
+class GreedyAlgorithm(Algorithm):
+    preferences = None
+
+    def fetch_preferences(self):
```

Comment:
If the method body stays the same like in Algorithm class, we don't need to override it